tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -9,11 +9,13 @@ var glob = require('glob')
 
 const configPaths = require('../../../config/paths.json')
 const lib = require('../../../lib/file-helper')
+const helperFunctions = require('../../../lib/helper-functions')
```

Comment:
Would it be worth importing just the function we need, so we can remove the namespacing?

```suggestion
const { componentNameToJavaScriptModuleName } = require('../../../lib/helper-functions')
```

Reply:
Good idea 👍